case_studies/strong_lensing/simulate.yaml

```diff
@@ -0,0 +1,118 @@
+
+variables :
+  output_dir : output_yaml
```

Comment:
My `output_yaml` directory doesn't appear contain any yaml. Could we just call it `output`?

Reply:
(It contains png and fits files)